.github/workflows/merge-autogenerated-content.yml

```
@@ -0,0 +1,45 @@
+name: Merge generated content updates
+
+# Trigger by cronjob
+on:
+  schedule:
+    # Merge the PR every Friday at 10:00 UTC
+    - cron: "0 10 * * FRI"
+  workflow_dispatch: {} # Allow manual triggering
+
+jobs:
+  merge-autogenerated-content:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Generate a token
+      id: generate-token
+      uses: actions/create-github-app-token@7bfa3a4717ef143a604ee0a99d859b8886a96d00 # v1.9.3
+      with:
+        app-id: ${{ vars.AUTO_MERGE_DOCS_PRS_APP }}
+        private-key: ${{ secrets.AUTO_MERGE_DOCS_PRS_APP }}
+
+    - name: Checkout code
+      uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 v4.1.2
```

Forgot the comment sign. My bad!

```suggestion
      uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
```